src/proxy-ui-api/src/main/resources/openapi-definition.yaml

```diff
@@ -2532,6 +2535,130 @@ paths:
           description: request specified an invalid format
         '500':
           description: internal server error
+  /endpoints/{id}/access-rights:
+    get:
+      tags:
+        - security server
+      summary: get access rights for the selected endpoint
+      operationId: getEndpointAccessRights
+      description: Administrator views endpoints access rights
+      parameters:
+        - in: path
+          name: id
+          description: id of the endpoint
+          required: true
+          schema:
+            type: string
+            format: text
+            minLength: 1
+            maxLength: 255
```

Comment:
maxLength is not really 255 since this maps to endpoint table PK which is BIGINT, which is -9223372036854775808 to 9223372036854775807.

So I guess maxLength(20) or maxLength(19) would be more correct. 20 if negatives are possible, I guess they could be.

I would probably either leave maxLength out (since Endpoint schema object does not define it either) or use maxLength(20), I can't see the argument for using the **wrong** value (even if it may have originally been selected by me sometime long ago). Maybe there is a possibility that we could be using a different database in some future scenario, but I doubt 255 would be certain to be correct in that case, either.

Also localGroup and serviceDescription endpoints suffer from the same.

Reply:
Removed maxLength